client/my-sites/current-site/notice.jsx

```diff
@@ -245,14 +112,13 @@ export class SiteNotice extends Component {
 	}
 
 	render() {
-		const { site, isMigrationInProgress, hasJITM } = this.props;
+		const { site, isMigrationInProgress } = this.props;
```

Comment:
`hasJITM` is no longer used in this component now, so I guess we can remove the line below as well:

https://github.com/Automattic/wp-calypso/blob/76e7c58d5385863f9c1a93469bc737e673c313fa/client/my-sites/current-site/notice.jsx#L170

Reply:
will tidy this up as a separate PR rather than re-open and delay this one